src/core/border_router/routing_manager.hpp

```diff
@@ -85,6 +85,14 @@ class RoutingManager : public InstanceLocator
      */
     otError Init(uint32_t aInfraIfIndex);
 
+    /**
+     * This method enables/disables the border routing manager.
+     *
+     * @param[in]  aEnabled  A boolean to enable/disable the routing manager.
+     *
+     */
+    void SetEnabled(bool aEnabled);
```

Comment:
is dynamically enable/disable necessary?

Reply:
The duckhorn use case doc requires the border router being configuration. I think for most cases, applications will not need this. But there needs a way to cease being a Border Route gracefully (remove prefixes before stop Thread), this is at least useful for testing. Without dynamical enable/disable, we need to wait minutes for the prefixes to be removed when a BR is stopped (we need to test with real time because there are non-thread devices, e.g. WiFi/Ethernet devices).